src/meta/proto-conv/tests/it/v033_table_meta.rs

```diff
@@ -0,0 +1,156 @@
+                ex::TableField::new("bool", ex::TableDataType::Boolean),
+                ex::TableField::new("int8", ex::TableDataType::Number(NumberDataType::Int8)),
+                ex::TableField::new("int16", ex::TableDataType::Number(NumberDataType::Int16)),
+                ex::TableField::new("int32", ex::TableDataType::Number(NumberDataType::Int32)),
+                ex::TableField::new("int64", ex::TableDataType::Number(NumberDataType::Int64)),
+                ex::TableField::new("uint8", ex::TableDataType::Number(NumberDataType::UInt8)),
+                ex::TableField::new("uint16", ex::TableDataType::Number(NumberDataType::UInt16)),
+                ex::TableField::new("uint32", ex::TableDataType::Number(NumberDataType::UInt32)),
+                ex::TableField::new("uint64", ex::TableDataType::Number(NumberDataType::UInt64)),
+                ex::TableField::new(
+                    "float32",
+                    ex::TableDataType::Number(NumberDataType::Float32),
+                ),
+                ex::TableField::new(
+                    "float64",
+                    ex::TableDataType::Number(NumberDataType::Float64),
+                ),
+                ex::TableField::new("date", ex::TableDataType::Date),
+                ex::TableField::new("timestamp", ex::TableDataType::Timestamp),
+                ex::TableField::new("string", ex::TableDataType::String),
+                ex::TableField::new("struct", ex::TableDataType::Tuple {
+                    fields_name: vec![s("foo"), s("bar")],
+                    fields_type: vec![ex::TableDataType::Boolean, ex::TableDataType::String],
+                }),
+                ex::TableField::new(
+                    "array",
+                    ex::TableDataType::Array(Box::new(ex::TableDataType::Boolean)),
+                ),
+                ex::TableField::new(
+                    "map",
+                    ex::TableDataType::Map(Box::new(ex::TableDataType::Boolean)),
+                ),
+                ex::TableField::new("variant", ex::TableDataType::Variant),
+            ],
+            btreemap! {s("a") => s("b")},
+        )),
+        catalog: "never-gonna-give-you-up".to_string(),
+        engine: "44".to_string(),
+        engine_options: btreemap! {s("abc") => s("def")},
+        storage_params: Some(StorageParams::default()),
+        part_prefix: "lulu_".to_string(),
+        options: btreemap! {s("xyz") => s("foo")},
+        default_cluster_key: Some("(a + 2, b)".to_string()),
+        cluster_keys: vec!["(a + 2, b)".to_string()],
+        default_cluster_key_id: Some(0),
+        created_on: Utc.with_ymd_and_hms(2014, 11, 28, 12, 0, 9).unwrap(),
+        updated_on: Utc.with_ymd_and_hms(2014, 11, 29, 12, 0, 10).unwrap(),
+        comment: s("table_comment"),
+        field_comments: vec!["c".to_string(); 21],
+        drop_on: None,
+        statistics: Default::default(),
+        shared_by: BTreeSet::new(),
```

Comment:
There should be some data in `shared_by` to assert a protobuf message can be correctly decoded.